### DISCUSSION TITLE
Blacklisted IPs

### DISCUSSION THREAD

**Discussion by elswork** [community/community#107441] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Dear GitHub team,

I hope you are fine. I am contacting you to inform you about an issue related to the anti-spam list. Four IP addresses have been included in this list and, as a result, are experiencing blocks or restrictions on access to their services.

The affected IP addresses are the following:

185.199.108.153
185.199.109.153
185.199.110.153
185.199.111.153

I understand that GitHub does not directly operate blacklisting, but I would like to request your assistance in resolving this issue. Could you please contact Spamhaus to review the list and take the necessary steps to remove these IP addresses from the anti-spam list?

Spamhaus is known for its efficiency in solving blacklist-related problems, and I am confident that they will be able to provide a quick and effective solution. I thank you in advance for your attention to this matter.

Sincerely,

Eloy

– GitHub user

**Comment by 0xdea** [comment]
See also https://github.com/orgs/community/discussions/106991

**Comment by sandeepkhadka123** [comment]
ohh..

**Comment by lcismondi** [comment]
I'm still having the same problem with the IPs. I saw the [other discussion](https://github.com/orgs/community/discussions/106991), but the problem continues, and I can't configure a new GitHub page. 

My DNS provider blocks the IPs when I try to create an "A" record with the following addresses:

- 185.199.108.153
- 185.199.109.153
- 185.199.110.153
- 185.199.111.153

On the other hand, I checked them on [Spamhaus](https://check.spamhaus.org/listed/?searchterm=185.199.108.153), and they are listed.
